layouts/partials/json-schema/resolve-additional-types.html

```diff
@@ -36,7 +38,11 @@
     {{/*
       Add the object we were passed into the $additional_objects array
     */}}
-    {{ $additional_objects = $additional_objects | append (partial "clean-object" $this_object) }}
+    {{ $this_additional_object := $this_object }}
+    {{ if not $is_root }}
+      {{ $this_additional_object = partial "clean-object" $this_object }}
+    {{ end }}
+    {{ $additional_objects = $additional_objects | append $this_additional_object }}
```

Comment:
`is_root` feels like it is going to be hard to use right and easy to mess up.

Two thoughts that occur to me:
 * I note that the the only place that `resolve-additional-types` is called *without* `is_root=true` (`get-additional-objects` below) calls `clean-object` on each of the returned objects anyway. Perhaps this call to `clean-object` is entirely redundant and can just be removed?
 * Failing that, can we call `clean-object` on the `schema` object *before* passing it into `resolve-additional-types`?

Reply:
Indeed the first suggestion works